src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -112,8 +90,81 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     selectPortfolioCryptoHumanBalanceByFilter(s, feeAssetBalanceFilter),
   )
 
+  const selectedCurrency = useAppSelector(selectSelectedCurrency)
+
+  const getEstimateFeesArgs: () => Promise<EstimateFeesInput> = useCallback(async () => {
+    const amountCryptoBaseUnit = bnOrZero(state?.deposit.cryptoAmount).times(
+      bn(10).pow(asset.precision),
+    )
+    const quote = await getThorchainSaversQuote(asset, amountCryptoBaseUnit)
+
+    return {
+      cryptoAmount: state?.deposit.cryptoAmount ?? '',
+      asset,
+      to: quote.inbound_address,
```

Comment:
thinking out loud - are we/how can we ensure against halt flags here, such that we don't deposit into a halted `inbound_address` that is liable to churn out?

Reply:
Abso-lutely: a81aca1d54c2af717145bf4ef2e94a24b776fc52
Will also give us double the safety, when this guy goes in https://github.com/shapeshift/web/blob/1a7dedbd7fef0f13fdd15c19f75d9d0fb0f48411/src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/types.ts#L44

Note this will just toast the regular error toast and doesn't bring UX changes yet - to be `toast`ed alongside expiry handling in a follow-up PR